### PR TITLE
Change downloads beta availability wording

### PR DESCRIPTION
**Pull request by renandincer**
@zaidf

### CODE REVIEW

File: products/stream/src/content/viewing-videos/download-videos.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ order: 5
 
 <Aside>
 
-Note: MP4 downloads are currently in beta and will be available to all users in coming weeks.
+Note: MP4 downloads are currently in open beta available to all Stream customers
```
Comment (kodster28) [inline products/stream/src/content/viewing-videos/download-videos.md]:
```suggestion
Note: MP4 downloads are currently in open beta and available to all Stream customers
```

**Review by kodster28 — APPROVED** [review]
Good from a content side, at least

**Review by Everlag — APPROVED** [review]
LGTM apart from @kodster28's comment.

**Comment by renandincer** [comment]
super, thank you @kodster28!

**Pull request merged**